src/Components/Resource/ListView.tsx

```diff
@@ -17,48 +12,34 @@ import { useTranslation } from "react-i18next";
 import { AdvancedFilterButton } from "../../CAREUI/interactive/FiltersSlideover";
 import CareIcon from "../../CAREUI/icons/CareIcon";
 import dayjs from "../../Utils/dayjs";
+import useQuery from "../../Utils/request/useQuery";
+import routes from "../../Redux/api";
 
 const Loading = lazy(() => import("../Common/Loading"));
 const PageTitle = lazy(() => import("../Common/PageTitle"));
 
 export default function ListView() {
-  const dispatch: any = useDispatch();
   const { qParams, Pagination, FilterBadges, advancedFilter, resultsPerPage } =
     useFilters({});
-  const [data, setData] = useState<any[]>([]);
-  const [totalCount, setTotalCount] = useState(0);
-  const [isLoading, setIsLoading] = useState(false);
   const { t } = useTranslation();
 
   const onBoardViewBtnClick = () =>
     navigate("/resource/board", { query: qParams });
   const appliedFilters = formatFilter(qParams);
 
   const refreshList = () => {
-    fetchData();
+    refetch();
   };
```

Comment:
You can simply call refetch directly instead of wrapping refetch inside refreshList right?